boiler-ctl

```diff
@@ -0,0 +1,149 @@
+#!/bin/sh
```

Comment:
Good stuff, we need to update README accoding to this? for better experience using the new CLI :thinking: 

(didn't test it intensively yet)